strongbox-web-core/src/test/resources/strongbox/routing/add-accept-rule-set.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@bhliva,

I think it would be better to have these test JSON resources as strings in the test cases.